logrotate.c

```diff
@@ -2978,6 +3017,10 @@ int main(int argc, const char **argv)
     poptFreeContext(optCon);
     nowSecs = time(NULL);
 
+    if (lockState(stateFile)) {
+        exit(3);
```

Comment:
Why exactly 3 was chosen as the exit code here?  This should be documented in any case.

Reply:
The exit values 1 and 2 are used, so I wanted to make it distinct.
But this is subject to changes and yes we should document it.